app/main.py

```diff
@@ -1,3 +1,71 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.shop import Shop
+from app.customer import Customer
+from app.car import Car
+
+
+def shop_trip() -> None:
```

Comment:
it is really long function. try to seperate this function to a few functions with one responsibility

Reply:
can add some methods to Customer class